docs/sources/mimir/tutorials/play-with-grafana-mimir/docker-compose.yml

```diff
@@ -35,7 +35,7 @@ services:
       - ../../../../../operations/mimir-mixin-compiled/dashboards/mimir-top-tenants.json:/var/lib/grafana/dashboards/mimir-top-tenants.json:ro
       - ../../../../../operations/mimir-mixin-compiled/dashboards/mimir-writes.json:/var/lib/grafana/dashboards/mimir-writes.json:ro
     ports:
-      - 9000:3000
+      - 3000:3000
```

Comment:
The first port should refer to the local mapping of the port, the second - to the port on the container. I was able to access the grafana instance on `localhost:9000` even though the grafana container is serving on `localhost:3000`. Did I misunderstand your PR description?

Reply:
I would expect it's to avoid clashing with an existing Grafana service. It happens to me frequently that I have to take down my Grafana service installed in the OS, to spin up a Docker Compose stack for testing.